tests/sentry/integrations/gitlab/test_repository.py

```diff
@@ -87,7 +87,7 @@ def create_repository(self, repository_config, integration_id, organization_slug
             json={'id': 99}
         )
 
-        with self.feature({'organizations:internal-catchall': True, 'organizations:repos': True}):
```

Comment:
I wrote this, but I don't see why it was ever added. @markstory it doesn't appear necessary. Just double checking that I'm not forgetting something obvious